src/js/components/Baani.js

```diff
@@ -153,6 +173,8 @@ export default class Baani extends React.PureComponent {
             key={shabad.id}
             id={`line-${shabad.id}`}
             className="line"
+            onMouseUp={this.makeSelection}
+            onMouseDown={this.removeSelection}
```

Comment:
Wouldn't work for split view then right?

Reply:
Yeah, it doesn't work in split view. Do we want it to work in split view?